src/content/en/updates/2017/09/webvr-origin-trial-chrome-62.md

```diff
@@ -0,0 +1,55 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Learn about what is changing in the WebVR API.
+
+{# wf_updated_on: 2017-09-28 #}
+{# wf_published_on: 2017-09-28 #}
+{# wf_tags: news,webvr #}
+{# wf_featured_snippet: Learn about what is changing in the WebVR API #}
+{# wf_blink_components: Blink>WebVR #}
+
+# WebVR changes in Chrome 62 {: .page-title }
+
+{% include "web/_shared/contributors/mscales.html" %}
+
+The current WebVR origin trial is ending on November 14,
+2017, shortly after the stable release of Chrome 62. We have begun a new trial with the WebVR 1.1
+API in Chrome 62 that will continue through Chrome 64.
+
+The new trial includes some API behavior updates that are consistent with the direction of the
+forthcoming [WebVR 2.0 spec][0]:
+
+- Use of WebVR is restricted in cross-origin iframes. If you intend for embedded cross-origin
+  iframes to be able to use WebVR, add the attribute allow="vr" to the iframe tag, or use a
+  [Feature-Policy][1] header ([spec discussion][2], [bug][3]).
+- Limit use of getFrameData() and submitFrame() to VRDisplay.requestAnimationFrame()
+  ([spec discussion][4], [bug][5]).
+- window.requestAnimationFrame() does not fire if the page is not visible, meaning it will not fire
+  on Android while WebVR is presenting ([spec discussion][6], [bug][7]).
+- The synthetic click event at viewport (0, 0) has been removed (for both Cardboard and the Daydream
+  controller touchpad) ([bug][8]). The vrdisplayactivate event is now considered a user gesture, and
+  may be used to request presentation and begin media playback, without relying on the click event.
+  Code that was previously relying on click event handlers for input should be converted to check
+  for gamepad button presses. ([Example implementation][9])
+- Chrome may exit presentation if the page takes greater than 5 seconds to display the first frame
+  ([code change][10]). It is recommended that the page display within 2 seconds and use a splash
+  screen if needed.
```

Comment:
Faulty parallelism. Do this:

It is recommended that the page display with two seconds and that a splash screen is used if needed.

Notice that both parts begin with 'that'.